test/gurka/test_route_incidents.cc

```diff
@@ -29,20 +29,35 @@ class IncidentsTest : public ::testing::Test {
 
     // connect the ways via the nodes
     const gurka::ways ways = {
-        {"AB", {{"highway", "tertiary"}}},  {"BC", {{"highway", "service"}}},
-        {"DEF", {{"highway", "primary"}}},  {"GHI", {{"highway", "primary"}}},
-        {"ADG", {{"highway", "motorway"}}}, {"BE", {{"highway", "secondary"}}},
-        {"EH", {{"highway", "tertiary"}}},
+        {"AB", {{"highway", "tertiary"}, {"driving_side", "left"}}},
```

Comment:
Had to add driving side since we're no longer using admin db and instead manually setting certain admin attributes.